src/as-validator.c

```diff
@@ -1689,6 +1689,12 @@ as_validator_check_release (AsValidator *validator, xmlNode *node, AsFormatStyle
 		/* Neither timestamp, nor date property exists */
 		if (timestamp == NULL)
 			as_validator_add_issue (validator, node, "release-time-missing", "date");
+		else {
+			/* check if the timestamp is both a number and higher than 3000. The 3000 is used to check, that it is not a year */
+			if (g_ascii_strtoll (timestamp, NULL, 10) < 3000) {
+				as_validator_add_issue (validator, node, "release-timestamp-invalid", "timestamp");
```

Comment:
The `hint` parameter should be the actual timestamp and not a "timestamp" string, so `as_validator_add_issue (validator, node, "release-timestamp-invalid", timestamp);`